maps/z2.dmm

```diff
@@ -57845,9 +57936,7 @@
 	},
 /area/centcom)
 "gAK" = (
-/obj/submachine/claw_machine,
-/turf/unsimulated/floor/wood/six,
-/area/afterlife/bar/sanctuary)
+)
```

Comment:
several sections of the map are missing turfs entirely

Reply:
not sure how the fuck that happened but it should be fixed